static/docs/get-started/example-pipeline.md

```diff
@@ -167,8 +167,12 @@ intermediate result.
 Second, you should see by now that the actual data is stored in the `.dvc/cache`
 directory, each file having a name in a form of an md5 hash. This cache is
 similar to Git's internal objects store but made specifically to handle large
-data files. DVC is using reflinks, hardlinks and other optimizations to manage
-your actual workspace without copying every time object from/to the cache.
+data files.
+
+> **Note!** For performance with large data files, DVC can use file links from
+the cache to the workspace to avoid copying actual file contents. Refer to
```

Comment:
did you run the pre-commit hook we have now? I think it puts `>` before each line

Reply:
I had not but just installed the pre-commit hook (which uses `prettier`) and it does add them indeed. BTW Rewrap also puts `>` before each line but only if a long comment has no line breaks yet, otherwise it doesn't...

Also, `prettier` re-wraps my paragraphs differently than VSCode Rewrap. (The pre-commit hook doesn't line-break links.)

So there's inconsistencies throughout the docs with these 2 things at least. Maybe I should just run `prettier` on everything?

Reply:
OK... I just ran ``npm bin`/prettier --write --prose-wrap=always` on all the files altered by this PR and the funny thing is that when `pre-commit` ran prettier again the results were different. So I'm not sure how to run `prettier` so it has the same settings as when it is ran by `pre-commit`.

Anyway, I noticed some problems with the auto-reformatting like that it changes how we wrote sidebar.json and it inserts more spaces to indent `yaml` code-blocks showing DVC-file examples so I'm turning it off for now. I'll ask about all this on Discord.